### PR TITLE
partial ingest codes

### PR DESCRIPTION
**Pull request by astro-c**
Y MKO data ingests

Update on #360

To do:
- [x] create lists for multiple instrumentation publications
- [x] finish manual ingest of publications
- [ ] pytest for ingest
- [x] pytest for photometry bands

### CODE REVIEW

**Comment by Ajb2307** [comment]
Hi!

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@
 .vscode
 .pytest_cache
 .ds_store
-bdnyc/
\ No newline at end of file
+bdnyc/
+/test.py
```
Comment (kelle) [inline .gitignore]: Tell me more about why you're editing the .gitignore.

**Review by kelle — COMMENTED** [review]
This is looking good! 

- I would prefer the ingest script be just one file. Basically, incorporate MKO move utils functions into the ingest script. 

- Next step will be to add tests

File: .gitignore (hunk 1 above)
Comment (astro-c) [reply .gitignore]: I was not meant to commit that change, sorry. It's a remote file I was using to test certain aspects of the code, I'll try to revert the commit ASAP.

**Comment by astro-c** [comment]
I believe I reverted the accidental commit of the .gitignore file and successfully merged the MKO utils with the MKO ingest

**Comment by astro-c** [comment]
Ingestion commit needs to be reversed. There were 2 things I had fixed with commits recently that I believed I had pushed but the push seems to have failed so the ingests are wrong.

**Comment by kelle** [comment]
Hey M, are you going to be able to finish this up or should we take it over. It's sooooo close!  Let us know.  Thanks!

**Comment by astro-c** [comment]
I can finish it but I need some tech support on this and I do not have much spare time available.

**Comment by kelle** [comment]
Could you say more about the problems you're having? It's probably best to use Slack.

Also, since your script is in good shape, I recommend considering starting fresh. 
- Make a copy of your scripts 
- start a fresh branch from upstream:main.
- re run the scripts.

We can discuss in more detail
in Slack and/or setup some zoom time to work together.

**Pull request closed without merging**